<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Effect</class>
 <widget class="QMainWindow" name="Effect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Effect</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Compressor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Overdrive</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QDial" name="dial"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Level</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QDial" name="dial_2"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Gain</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QDial" name="dial_3"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_3"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>52</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QDial" name="dial_4"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_4"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QDial" name="dial_5"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_5"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>52</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>ON</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>za wzmakiem</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
